docs/run-platform-server/scaling.mdx

```diff
@@ -23,7 +23,7 @@ For low to medium load environments with up to 30-90 days of data history retent
 
 ![](/assets/horizon-scaling/Topology-2VMs.png)
 
-### Extension: Isolating Captive Core
+### Isolating Captive Core
```

Comment:
We have deprecated remote captive core so I don't think it's possible to isolate captive core to its own VM

Reply:
great catch, removed, thanks!